test.scss

```diff
@@ -0,0 +1,3 @@
+* {
+  color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.